### PR TITLE
Convert return usage values from Mongo + add simple usages do Default WH when error

### PR DESCRIPTION
**Pull request by damian1976**
Convert return usage values to conform to new files layout and classes division + removes the error when Warehouse does not exists when pulling Openstack simple usages (usages are added to Default WH)

### CODE REVIEW

File: src/ralph_scrooge/plugins/cost/support.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def _costs(
         }
         """
         logger.info("Calculating supports costs")
-        support_type = ExtraCostType.objects.get(pk=2)  # from fixture
+        support_type = ExtraCostType.objects_admin.get(pk=2)  # from fixture
```
Comment (mkurek) [inline src/ralph_scrooge/plugins/cost/support.py]: here only `objects` should be considered

File: src/ralph_scrooge/plugins/cost/team.py [hunk 2]
```diff
@@ -216,7 +216,7 @@ def _get_cores_usage_type(self):
         """
         Physical CPU cores usage type definition
         """
-        return UsageType.objects.get_or_create(
+        return UsageType.objects_admin.get_or_create(
```
Comment (mkurek) [inline src/ralph_scrooge/plugins/cost/team.py]: ditto

File: src/ralph_scrooge/rest/allocationclient.py [hunk 3]
```diff
@@ -48,7 +48,7 @@ def _get_service_usage_type(
     ):
         service = Service.objects.get(id=service)
         try:
-            pricing_service = PricingService.objects.get(services=service)
+            pricing_service = PricingService.objects_admin.get(services=service)
```
Comment (mkurek) [inline src/ralph_scrooge/rest/allocationclient.py]: only `objects`

File: src/ralph_scrooge/rest/allocationclient.py [hunk 4]
```diff
@@ -241,7 +241,7 @@ def post(self, request, year, month, service, env, *args, **kwargs):
                 service__id=service,
                 environment__id=env,
             )
-            other_type = ExtraCostType.objects.get(id=1)
+            other_type = ExtraCostType.objects_admin.get(id=1)
```
Comment (mkurek) [inline src/ralph_scrooge/rest/allocationclient.py]: only `objects`

**Comment by mkurek** [comment]
:ok:

File: src/ralph_scrooge/plugins/collect/_openstack_base.py [hunk 5]
```diff
@@ -206,7 +206,7 @@ def run_plugin(self, sites, today, **kwargs):
                 logger.error('Invalid warehouse: {}'.format(
                     site['WAREHOUSE']
                 ))
-                continue
+                warehouse = Warehouse.objects.get(pk=1)
```
Comment (ghost) [inline src/ralph_scrooge/plugins/collect/_openstack_base.py:210]: Add comment what is the magic '1', please.

Comment (damian1976) [inline src/ralph_scrooge/plugins/collect/_openstack_base.py:210]: '1' means 'Default' warehouse

Comment (mkurek) [inline src/ralph_scrooge/plugins/collect/_openstack_base.py:210]: yes - but when someone from 'outside' will look into this code he won't know this - just add some comment above this line where does this 1 come from.

**Comment by ar4s** [comment]
:ok:

**Pull request merged**